src/sentry/tasks/post_process.py

```diff
@@ -976,6 +977,20 @@ def process_workflow_engine(job: PostProcessJob) -> None:
     with sentry_sdk.start_span(op="tasks.post_process_group.workflow_engine.process_workflow"):
         process_workflows(workflow_event_data)
 
+    try:
+        process_workflows_event.delay(
```

Comment:
thoughts on adding a feature flag here so we can gate which way we enter the workflow engine? mostly just incase there are any initial rollout issues.